src/NodeAnalyzer/FuncCallAnalyzer.php

```diff
@@ -0,0 +1,30 @@
+<?php declare(strict_types=1);
+
+namespace Rector\NodeAnalyzer;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Name;
+
+/**
+ * Read-only utils for FuncCall Node:
+ * "someMethod()"
+ */
+final class FuncCallAnalyzer
+{
+    /**
+     * Checks "specificFunction()"
+     */
+    public function isName(Node $node, string $name): bool
```

Comment:
This is awesome. I’ll provide some refactorings across our Rectors :)

Reply:
I tried to do that already, but there might be some cases I missed :)